filebeat/module/system/syslog/test/darwin-syslog-sample.log-expected.json

```diff
@@ -1,90 +1,93 @@
 [
-  {
-    "_index" : "filebeat-2017.01.27",
-    "_type" : "log",
-    "_id" : "AVngMBcpUmQau8zc_UuM",
-    "_score" : 1.0,
-    "_source" : {
-      "@timestamp" : "2017-12-13T11:35:28.000Z",
-      "system" : {
-        "syslog" : {
-          "hostname" : "a-mac-with-esc-key",
-          "pid" : "21412",
-          "program" : "GoogleSoftwareUpdateAgent",
-          "message" : "2016-12-13 11:35:28.420 GoogleSoftwareUpdateAgent[21412/0x700007399000] [lvl=2] -[KSAgentApp updateProductWithProductID:usingEngine:] Checking for updates for \"All Products\" using engine <KSUpdateEngine:0x100341a00\n\t\tticketStore=<KSPersistentTicketStore:0x100204520 store=<KSKeyedPersistentStore:0x100213290\n\t\t\tpath=\"/Users/tsg/Library/Google/GoogleSoftwareUpdate/TicketStore/Keystone.ticketstore\"\n\t\t\tlockFile=<KSLockFile:0x1002160e0\n\t\t\t\tpath=\"/Users/tsg/Library/Google/GoogleSoftwareUpdate/TicketStore/Keystone.ticketstore.lock\"\n\t\t\t\tlocked=NO\n\t\t\t>\n\t\t>>\n\t\tprocessor=<KSActionProcessor:0x1003bb5f0\n\t\t\tdelegate=<KSUpdateEngine:0x100341a00>\n\t\t\tisProcessing=NO actionsCompleted=0 progress=0.00\n\t\t\terrors=0 currentActionErrors=0\n\t\t\tevents=0 currentActionEvents=0\n\t\t\tactionQueue=( )\n\t\t>\n\t\tdelegate=(null)\n\t\tserverInfoStore=(null)\n\t\terrors=0\n\t>",
-          "timestamp" : "Dec 13 11:35:28"
+    {
```

Comment:
I actually didn't have to update them. It just passed without changes with the old `type` inside. But it bugged me to have "outdated" docs in the tests so I updated them manually. Then my editor automatically did the formatting. Changes are only in whitespaces and `input_type` to `prospector.type`. I can also revert it to master if you prefer.